include/zephyr/bluetooth/bluetooth.h

```diff
@@ -2125,6 +2125,16 @@ struct bt_le_scan_cb {
 					   BT_GAP_SCAN_FAST_INTERVAL, \
 					   BT_GAP_SCAN_FAST_WINDOW)
 
+/**
+ * @brief Helper macro to enable active scanning to discover new devices with window == interval.
+ *
+ * Continuous scanning should be used to maximize the chances of receiving advertising packets.
+ */
+#define BT_LE_SCAN_ACTIVE_CONTINUOUS BT_LE_SCAN_PARAM(BT_LE_SCAN_TYPE_ACTIVE, \
+					   BT_LE_SCAN_OPT_FILTER_DUPLICATE, \
+					   BT_GAP_SCAN_FAST_INTERVAL, \
+					   BT_GAP_SCAN_FAST_INTERVAL)
```

Comment:
Some indentation seems off here

Initially I was not fully on board with the name "CONTINUOUS", but I see that's the term the core spec uses: 

> The LE_Scan_Window parameter shall always be set to a value smaller or equal to the value set for the LE_Scan_Interval parameter. If they are set to the same value scanning should be run continuously.

Reply:
Instead of using `BT_GAP_SCAN_FAST_INTERVAL` define `BT_GAP_SCAN_FAST_WINDOW` of 30 ms.... will correspond to  `TGAP(scan_fast_window)` recommended value.